lib/bap_disasm/disasm.cpp

```diff
@@ -124,7 +124,7 @@ class disassembler {
         };
     }
 
-    void set_memory(int64_t addr, const char *data, int offset, int length) {
+    void set_memory(int64_t addr, const uint8_t *data, int offset, int length) {
```

Comment:
Ok. These changes are pretty directly tied to llvm 3.8 changes though, which is why I included them in this PR. Additionally, these changes are necessary for new llvm plugin